src/Groups/group_characters.jl

```diff
@@ -1207,7 +1229,7 @@ Base.iterate(chi::GAPGroupClassFunction, state = 1) = state > length(chi.values)
 
 @doc Markdown.doc"""
     degree(::Type{T} = QQFieldElem, chi::GAPGroupClassFunction)
-           where T <: Union{IntegerUnion, ZZRingElem, mpq, QQAbElem}
+           where T <: Union{IntegerUnion, ZZRingElem, QQFieldElem, QQAbElem}
```

Comment:
Not related to this PR, but why is this not `ZZRingElem` the default?

Reply:
`GAPGroupClassFunction` objects are more general than (virtual) characters, one can think of scalar multiples of differences of characters. One example where rational multiples occur in practice are the coefficient vectors of idempotents in the centre of the group algebra.
Of course one can argue that calling `degree` for such a class function does not make sense. Or one can argue that one could consider arbitrary cyclotomics for scalar multiplication of class functions.